app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -3836,14 +3849,26 @@ class BrowserTabViewModel @Inject constructor(
     }
 
     fun onDuckChatMenuClicked() {
+        openDuckChat(DuckChatPixelName.DUCK_CHAT_OPEN_BROWSER_MENU)
+    }
+
+    fun onDuckChatOmnibarButtonClicked(query: String?) {
+        openDuckChat(DuckChatPixelName.DUCK_CHAT_OPEN_URL_FOCUS, query)
```

Comment:
This pixel was triaged in https://app.asana.com/0/1208671518894266/1209195479842604 but wasn't added yet since the new button work was in-progress. I opted to add it here.